### ISSUE TITLE
`Application is not in valid state to perform this operation`

### ISSUE THREAD

**Issue by ptbrowne** [cozy/cozy-stack#680]
It seems that after issues with CouchDB, it is not possible to get a konnector in a good state.

```
$ ssh recette.int.cozycloud.cc  'cozy-stack --domain recette.cozy.works konnectors update edf'
16:17 patrick: time="2017-06-05T14:16:37Z" level=error msg="CouchDB(no_couch): could not create connection with the server - Put http://xxxxxxxxx@ha-couch-1-int:5984/recette-cozy-works%2Fio-cozy-konnectors/io.cozy.konnectors%2Fedf: net/http: request canceled (Client.Timeout exceeded while awaiting headers)"
$ ssh recette.int.cozycloud.cc  'cozy-stack --domain recette.cozy.works konnectors update edf'
16:17 patrick: time="2017-06-05T14:16:42Z" level=error msg="Application is not in valid state to perform this operation"
$ ssh recette.int.cozycloud.cc  'cozy-stack --domain recette.cozy.works konnectors update edf'
16:17 patrick: time="2017-06-05T14:16:45Z" level=error msg="Application is not in valid state to perform this operation"
$ ssh recette.int.cozycloud.cc  'cozy-stack --domain recette.cozy.works konnectors uninstall edf'
16:17 patrick: time="2017-06-05T14:16:52Z" level=error msg="Unqualified error: Application is not in valid state to perform this operation"
```

**Comment by nono** [comment]
The konnector is stuck in `upgrading` state:

```
$ cozy-stack --domain recette.cozy.works konnectors ls
bouyguesbox	git://github.com/cozy/cozy-konnector-bouyguesbox.git#build	ready
bouyguestelecom	git://github.com/cozy/cozy-konnector-bouyguestelecom.git#build	ready
edf	git://github.com/cozy/cozy-konnector-edf.git#build	upgrading
```

**Comment by nono** [comment]
Fixed by #687
